### DISCUSSION TITLE
可不可以增加一个截长屏的功能 类似我贴的这个网址里这个软件似的 iShot

### DISCUSSION THREAD

**Discussion by reliXQ** [Snipaste/feedback#2091] [Ideas]
我觉得他的截长屏效果挺好的,能不能移植到Snipaste上
iShot  这个软件是Mac上使用的 可以看下他是怎么实现截长屏的,有点意思
https://www.better365.cn/ishot.html?bd_vid=8644034705118571170

**Comment by Geeyun-JY3** [comment]
#19
